### DISCUSSION TITLE
Removing all checkpoints doesn't update them in the connected org

### DISCUSSION THREAD

**Discussion by randi274** [forcedotcom/salesforcedx-vscode#5812] [Ideas]
### Summary

When I have been working with a few checkpoints, then remove them in VS Code, they're not automatically removed in the org during the easy debugging flow (Debug Test / Launch Apex Replay Debugger with the Current File)

### Steps To Reproduce:

1.  Toggle a few checkpoints and either run the Update Checkpoints command to update them in the org, debug a test, or use Launch Apex Replay Debugger with the Current File.
1.  Remove all checkpoints in VS Code (Toggle Checkpoint, or delete by clicking them directly)
1.  Debug at Test or Launch Apex Replay Debugger with the Current File
1.  Notice that Update Checkpoints didn't run; code will still stop at the checkpoints with the replay debugger even though they don't exist in the UI anymore. If you open the Checkpoint Viewer in the Dev Console, you'll see they're still there as well.

### Expected result

When running through Debug Test or Launch Apex Replay Debugger with Current File, checkpoints are removed. 

### Actual result

Checkpoints are not removed because VS Code thinks it doesn't have any checkpoints to work with.

### Additional information

**Salesforce Extension Version in VS Code**: 54.8.0
**OS and version**: Mac 11.6.5

**Comment by randi274** [comment]
@AnanyaJha I wasn't quite willing to call this a dev bug without your review; if this is WAD we can update docs instead.

**Comment by AnanyaJha** [comment]
@randi274 I would say this is WAD, you would need to run Update Checkpoints again for the checkpoints to be removed from the org afair
